### PR TITLE
Adding a template for pull request workflow

### PR DESCRIPTION
**Pull request by s-tuli**

### CODE REVIEW

File: .github/workflows/pullrequest.yml.template [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+#
+
+name: PRFlow <Name of the Workflow>
+
+on: 
+  pull_request:
+    branches:
+    - master
+    paths:
+    - <Path to the Service>/**
```
Comment (greenie-msft) [inline .github/workflows/pullrequest.yml.template]: Can we call out the template variables more? At a first glance, it's hard to see where they are.

Maybe something like (Capitalize the placeholder): 
<PATH_TO_THE_SERVICE>

File: .github/workflows/pullrequest.yml.template [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Name of the PlaceHolder                 -  Description
+########################################################################################################################
+# <Name of the WorkFlow>                  - Workflow name. example: Bikes
+# <Path to the service>                   - Path to the service where the developer would like to trigger the Flow/ action on pull request - Example: samples/BikeSharingApp/Bikes.
```
Comment (greenie-msft) [inline .github/workflows/pullrequest.yml.template]: This is really the path to the service that must be modified in order for the workflow to trigger.

File: .github/workflows/pullrequest.yml.template [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Name of the PlaceHolder                 -  Description
+########################################################################################################################
+# <Name of the WorkFlow>                  - Workflow name. example: Bikes
+# <Path to the service>                   - Path to the service where the developer would like to trigger the Flow/ action on pull request - Example: samples/BikeSharingApp/Bikes. 
+# <Path to the Dockerfile>                - Path to the folder which contains the Dockerfile
```
Comment (greenie-msft) [inline .github/workflows/pullrequest.yml.template]: Path to the directory that contains the DockerFile for the modified service.

File: .github/workflows/pullrequest.yml.template [hunk 4]
```diff
@@ -0,0 +1,76 @@
+# Name of the PlaceHolder                 -  Description
+########################################################################################################################
+# <Name of the WorkFlow>                  - Workflow name. example: Bikes
+# <Path to the service>                   - Path to the service where the developer would like to trigger the Flow/ action on pull request - Example: samples/BikeSharingApp/Bikes. 
+# <Path to the Dockerfile>                - Path to the folder which contains the Dockerfile
+# <Name of the Service>                   - Pass this as a parameter for multiple steps below. Lower Cased Name of the Service, for example: bikes
+# <Path to the Charts for that service>   - Pass this in as a parameter for the Helm Install PR step
```
Comment (greenie-msft) [inline .github/workflows/pullrequest.yml.template]: Path to the service's helm charts

File: .github/workflows/pullrequest.yml.template [hunk 5]
```diff
@@ -0,0 +1,76 @@
+# Name of the PlaceHolder                 -  Description
+########################################################################################################################
+# <Name of the WorkFlow>                  - Workflow name. example: Bikes
+# <Path to the service>                   - Path to the service where the developer would like to trigger the Flow/ action on pull request - Example: samples/BikeSharingApp/Bikes. 
+# <Path to the Dockerfile>                - Path to the folder which contains the Dockerfile
+# <Name of the Service>                   - Pass this as a parameter for multiple steps below. Lower Cased Name of the Service, for example: bikes
+# <Path to the Charts for that service>   - Pass this in as a parameter for the Helm Install PR step
+#                                           This is the path to the folder where the charts are present for that service. example: samples/BikeSharingApp/Bikes/charts/bikes 
+# <ReleaseName>                           - Pass this in as a parameter for the Helm Install PR step
```
Comment (greenie-msft) [inline .github/workflows/pullrequest.yml.template]: I would be more descriptive about why this is required. As an example. what is it used for?

File: .github/workflows/pullrequest.yml.template [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+########################################################################################################################
+# <Name of the WorkFlow>                  - Workflow name. example: Bikes
+# <Path to the service>                   - Path to the service where the developer would like to trigger the Flow/ action on pull request - Example: samples/BikeSharingApp/Bikes. 
+# <Path to the Dockerfile>                - Path to the folder which contains the Dockerfile
+# <Name of the Service>                   - Pass this as a parameter for multiple steps below. Lower Cased Name of the Service, for example: bikes
+# <Path to the Charts for that service>   - Pass this in as a parameter for the Helm Install PR step
+#                                           This is the path to the folder where the charts are present for that service. example: samples/BikeSharingApp/Bikes/charts/bikes 
+# <ReleaseName>                           - Pass this in as a parameter for the Helm Install PR step
+# 
+# Note: This template assumes that you use Helm. If not, replace the Helm Install PR step with your own step
```
Comment (greenie-msft) [inline .github/workflows/pullrequest.yml.template]: Maybe bring this up to the top of the template?

File: .github/workflows/pullrequest.yml.template (hunk 1 above)
Comment (s-tuli) [reply .github/workflows/pullrequest.yml.template]: Updated.

File: .github/workflows/pullrequest.yml.template (hunk 2 above)
Comment (s-tuli) [reply .github/workflows/pullrequest.yml.template]: Updated.

File: .github/workflows/pullrequest.yml.template (hunk 3 above)
Comment (s-tuli) [reply .github/workflows/pullrequest.yml.template]: Updated.

File: .github/workflows/pullrequest.yml.template (hunk 4 above)
Comment (s-tuli) [reply .github/workflows/pullrequest.yml.template]: Updated

File: .github/workflows/pullrequest.yml.template (hunk 5 above)
Comment (s-tuli) [reply .github/workflows/pullrequest.yml.template]: Added details for <RELEASE_NAME>

File: .github/workflows/pullrequest.yml.template (hunk 6 above)
Comment (s-tuli) [reply .github/workflows/pullrequest.yml.template]: Yes. I agree. Updated.

**Pull request merged**